platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: "Overview"
+description: "Data Studios public preview."
+date: "1 Oct 2024"
+tags: [data, studios]
+---
+
+Data Studios is a unified platform where you can host a combination of container images and compute environments for interactive tertiary analysis using your preferred tools, like JupyterLab and RStudio Notebooks, Visual Studio Code IDEs, or Xpra remote desktops. Each data studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.
+
+On Seqera Cloud, the free tier permits only one running data studio session at a time. To run simultaneous data studios, [contact Seqera][contact] for a Seqera Cloud Pro license.
+
+:::note
+Data Studios is currently in **public preview** and is available from Seqera Platform v24.1. If you experience any problems during the deployment process please contact your account executive. Data Studios in Enterprise is not enabled by default. You can enable Data Studios in the [environment variables configuration](../enterprise/data-studios.mdx).
+:::
+
+Select the **Data Studios** tab in Platform to view all data studios. The list includes the name, cloud provider, analysis template, region, author, creation date, and the status of each session. In this view, you can add a new data studio session, as well as start, stop, and connect to an existing data studio session. You can dynamically filter the list of data studio sessions using the search bar and searching by name (default), author username, or compute environment name. Selecting the session name opens a detailed view of the data studio, displaying configuration information.
+
+## Requirements
```

Comment:
Shouldn't this go a bit higher?

Reply:
Is there a reason this should go higher? Prior to this we're only intro'ing what Data Studios _is_. This is the first paragraph after that, which discusses what it looks like in the Platform UI. I think it should stay here unless there's a compelling reason to move it.

Reply:
It just read a bit strangely to me. Like I felt like I'd already started doing things and then you stop me to talk about prerequisites.

Reply:
I see why, though. It is kind of dangling text; it requirements comes first, it needs its own section or to be shoehorned into a different section, since it's kind of a "here's where you find DS in the UI" explanation.